geode-lucene/src/main/java/org/apache/geode/cache/lucene/internal/LuceneEventListener.java

```diff
@@ -53,6 +54,8 @@
   Logger logger = LogService.getLogger();
 
   private final RepositoryManager repositoryManager;
+  private CountDownLatch isFileAndChunkRegionReady = new CountDownLatch(1);
+  private boolean ready = false;
```

Comment:
Does this need to be a separate flag? Seems like the CoutDownLatch already has the flag. If not this should be volatile.